site/components/pages/Pledge/PledgeForm/index.tsx

```diff
@@ -52,10 +54,14 @@ export const PledgeForm: FC<Props> = (props) => {
   const onSubmit: SubmitHandler<PledgeFormValues> = async (
     values: PledgeFormValues
   ) => {
+    const nonce = values.nonce.toString();
+    const signature = (await signer?.signMessage(nonce)) as string;
```

Comment:
We can customise what message to show users when we prompt them to sign to edit.
Anyone we can ask and/or run the copy by?